cpp/include/cudf/strings/char_types/char_types.hpp

```diff
@@ -26,6 +26,9 @@ namespace strings
 /**
  * @brief Character type values.
  * These types can be or'd to check for any combination of types.
+ *
+ * This cannot be turned into an enum class because or'd entries can
+ * result in values that are not in the class.
```

Comment:
I'm not sure what this means. Can you provide an example? Just that not all combinations of bits has a meaning?

Reply:
There are many combinations that have meaning. If you want to check for types NUMERIC and SPACE then you could specify NUMERIC|SPACE. If this were a class enum then this combination could not map to any existing class item. Not sure if that helps.

Reply:
Yes it does. I have never tried to `&` or `|` class enums, but it makes sense that it would not work.